packages/samples/headless-react/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "headless-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@coveo/headless": "*",
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.3",
+    "@testing-library/user-event": "^12.6.2",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^12.19.15",
+    "@types/react": "^16.14.2",
+    "@types/react-dom": "^16.9.10",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
+    "typescript": "^4.1.3",
+    "web-vitals": "^0.2.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
`npm start` on the root directory appears to now also launch the react sample, is this intentional?

Reply:
Hmm, I see, I don't think it should, I will switch the command to `dev`.